source/php/Module/EventForm/views/components/fields/radio.blade.php

```diff
@@ -1,14 +1,28 @@
-@foreach($field['options'] as $value => $label)
-    @option([
-        'type' => 'radio',
-        'checked' => $loop->index === 0,
-        'attributeList' => [
-            'data-id' => $option->id,
-            'name' => $field['name'],
-            'value' => $value,
-        ],
-        'classList' => ['u-display--inline-block'],
-        'label' => $label
-    ])
-    @endoption
-@endforeach
+<div class="c-field">
+    @if(!empty($field['label']))
+        <div class="c-field__label">
+            {!! $field['label'] !!}
```

Comment:
same question as above again.
Also can't we make the label and description a partial instead so we don't duplicate code?

Reply:
Its not escaped so that simple HTML can be added to labels and descriptions. For example <strong> or <br> which have been used in some places. Fixed so that helpers and labels are in separate file now. Would probably be best to add labels and descriptions to all components in the component lib though.